cmd/contour/serve.go

```diff
@@ -707,13 +707,22 @@ func getDAGBuilder(ctx *serveContext, clients *k8s.Clients, clientCert, fallback
 		responseHeadersPolicy.Remove = append(responseHeadersPolicy.Remove, ctx.Config.Policy.ResponseHeadersPolicy.Remove...)
 	}
 
+	var requestHeadersPolicyIngress dag.HeadersPolicy
```

Comment:
Still missing the actual `headersPolicyService` in `ingress_processor.go` to apply the headers to the clusters. Will wrap that up tomorrow.

Reply:
Looks like this is done now.